spec/requests/custom_actions_spec.rb

```diff
@@ -75,7 +75,7 @@ def expect_result_to_have_custom_actions_hash
       get api_service_url(nil, svc1)
 
       expect_result_to_have_keys(%w(id href actions))
-      expect(response.parsed_body["actions"].collect { |a| a["name"] }).to match_array(%w(edit add_resource remove_resource remove_all_resources add_provider_vms))
+      expect(response.parsed_body["actions"].collect { |a| a["name"] }).to match_array(%w(edit edit edit add_resource remove_resource remove_all_resources add_provider_vms))
```

Comment:
Can you apply the same change here too ?  only look at post actions.  otherwise, looks good. Thanks.